service-front/app/src/Viewer/src/Handler/PaperVerification/LpaReadyToViewHandler.php

```diff
@@ -53,30 +51,12 @@ public function handle(ServerRequestInterface $request): ResponseInterface
 
     public function handleGet(ServerRequestInterface $request): ResponseInterface
     {
-        $code = $this->state($request)->lpaCode ?? 'P-AB12-CD34-EF56-G7';
-        //$surname = $this->state($request)->donorSurname ?? 'Babara Gilson';
-
-        if (isset($code)) {
-            // TODO - LPA service call to check lpa match
-            //$lpa = $this->lpaService->getLpaByLpaCode($this->code, $surname, null);
-
-            // mocking Lpa data for testing page
-            $lpa = json_decode(
-                file_get_contents(
-                    __DIR__ . '../../../../../../test/fixtures/combined_lpa.json'
-                ),
-                true
-            );
-
-            $combinedSiriusLpa = ($this->lpaDataFormatter)($lpa);
-            $donor             = $combinedSiriusLpa->getDonor()->getFirstname() .
-                ' ' . $combinedSiriusLpa->getDonor()->getSurname();
-        }
+        // TODO: remove hardcoded donor name and lpa type from state and pass it to twig template
 
         return new HtmlResponse($this->renderer->render(self::TEMPLATE, [
             'form'       => $this->form->prepare(),
-            'donor_name' => $donor,
-            'lpa_type'   => $combinedSiriusLpa->getLpaType(),
+            'donor_name' => $this->state($request)->donorName ?? '(Donor name to be displayed here)',
```

Comment:
```suggestion
            'donor_name' => $this->state($request)->donorName ?? throw new ExpectedValueNotInStateException(),
```

It's possible to do this now in PHP 8.3 and I think is a better approach to values being missing that we expect to be there. Potentially you'd capture this error in the abstract class handle method and maybe show something? Alternatively erase the workflow and start again?